config/environments/production.rb

```diff
@@ -82,23 +73,17 @@
   # the I18n.default_locale when a translation cannot be found).
   config.i18n.fallbacks = true
 
-  # Send deprecation notices to registered listeners.
-  config.active_support.deprecation = :notify
+  # Don't log any deprecations.
+  config.active_support.report_deprecations = false
```

Comment:
They seem to have change their minds since 6.1 :see_no_evil: I don't mind though since we aren't usually scrutinising the prod logs